context-free.md

```diff
@@ -2,15 +2,19 @@
 
 ##  Who is this programming language for?
 
+Context Free is essentially a computer programming language with potential utilisation in graphic design owing to its ability to produce intricate images in terms of variations of simple shapes: squares, triangles, and circles. A few lines of code of skewing, rotating, transforming and flipping these basic shapes as well as using complex programming concepts such as loops and recursion enables highly-detailed, organised, and specific designs to be created. In my view,  this language is used to serve those users who wish to create intricate and detailed patterns such as fractals, since these are easier to generate with a greater level of accuracy through programming statements rather than manually using Photoshop or another photo editor.
```

Comment:
Yes, I agree with the point you made when compared to Photoshop. Context Free is much easier and more efficient than other photo editors when it comes complex graph but with patterns in it.